pilot/pkg/security/authn/v1beta1/policy_applier.go

```diff
@@ -216,6 +217,12 @@ func NewPolicyApplier(rootNamespace string,
 	}
 }
 
+// Remove all Non-Alphanumeric Characters from a jwksURI and append to "n"
```

Comment:
the comment is out-dated.

Reply:
Sure. then do I need to comment the base64 encoding here?

Reply:
No need, but you can comment inside the function about why it uses base64 instead of using jwksURI directly.

Reply:
Fixed. Thanks a lot!